bin/feature

```diff
@@ -339,8 +339,7 @@ when 'pull'
    Git::run_safe(["git fetch"])
 
    current = Git::current_branch
-   upstream = "#{current}@{upstream}"
-   upstream_hash = Git::branch_hash(upstream)
+   upstream_hash = `git rev-parse #{current}@{upstream}`.strip
```

Comment:
I think this doesn't do everything that `Git::branch_hash` does..

dev_block :sunny: on maybe reverting this particular change.